hooks/removeNotRelevantParts.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = register => {
+    register('generate:after', generator => {
+        let hasMqtt = false;
```

Comment:
nitpick: you can just have `let hasMqtt` as `if (!hasMqtt)` condition will anyway resolve to `true` 

so you can just have `let hasMqtt, hasAmqp, hasKafka`

Reply:
Thanks! JS is not primary language, so i write it a little bit straightforward :sweat_smile:
Please check a new version

Reply:
no worries, man, happy to give hints, at least I can help like this